tsl/src/chunk_api.c

```diff
@@ -1672,3 +1670,24 @@ chunk_create_empty_table(PG_FUNCTION_ARGS)
 
 	PG_RETURN_BOOL(true);
 }
+
+#define CREATE_CHUNK_TABLE_NAME "create_chunk_table"
+
+void
+chunk_api_call_create_empty_chunk_table(const Hypertable *ht, const Chunk *chunk,
```

Comment:
I am wondering if this code doesn't belong in the other function or `tsl/src/chunk.c`?

In most cases, `chunk_api.c` implements the SQL function API (PG_FUNCTIONS) that is called on the data node side, not the call-out code on the access node.

Reply:
Remotely calling `create_table` is also done here, so it was the main reason to place it here. Also I use few static functions from this file.

Reply:
And, yes, I first thought to have the complete implementation in `chunk.c`.